src/android/activity/MainActivity.java

```diff
@@ -1,59 +1,48 @@
 package com.usabilla;
 
-import android.app.FragmentTransaction;
-import android.content.BroadcastReceiver;
-import android.content.Context;
 import android.content.Intent;
-import android.content.IntentFilter;
 import android.graphics.Color;
 import android.os.Build;
 import android.os.Bundle;
-import android.widget.Toast;
 
 import androidx.annotation.NonNull;
 import androidx.appcompat.app.AppCompatActivity;
-import androidx.localbroadcastmanager.content.LocalBroadcastManager;
+import androidx.fragment.app.FragmentTransaction;
 
-import com.usabilla.sdk.ubform.UbConstants;
-import com.usabilla.sdk.ubform.sdk.entity.FeedbackResult;
+import io.cordova.hellocordova.R;
 import uk.co.reallysmall.cordova.plugin.fragment.CordovaFragment;
 
 public class MainActivity extends AppCompatActivity {
 
-    private IntentFilter closeCampaignFilter = new IntentFilter(UbConstants.INTENT_CLOSE_CAMPAIGN);
-    private BroadcastReceiver receiverCampaignClosed;
-
     public CordovaFragment currentFragment;
+    public FakeR fakeR;
```

Comment:
Why have we extracted it? Do we use it anywhere else than in the `OnCreate`

Reply:
nope we only use it on on create

Reply:
So we could move it in there to be a local field

Reply:
done